x-pack/plugins/ml/public/application/routing/routes/explorer.tsx

```diff
@@ -231,15 +231,24 @@ const ExplorerUrlStateManager: FC<ExplorerUrlStateManagerProps> = ({ jobsWithTim
       : undefined;
 
   useEffect(() => {
+    /**
+     * For the "View by" swim lane the limit is the cardinality of the influencer values,
+     * which is known after the initial fetch.
+     * When looking up for top influencers for selected range in Overall swim lane
+     * the result is filtered by top influencers values, hence there is no need ot set the limit.
```

Comment:
Nit - typo `ot` should be `to`

Reply:
Fixed in https://github.com/elastic/kibana/pull/84258/commits/c073125e13e1da15ac3f77325227fb68d582c0ab